src/ast/nodes/ImportExpression.ts

```diff
@@ -41,6 +42,12 @@ interface DynamicImportMechanism {
 export default class ImportExpression extends NodeBase {
 	declare options: ExpressionNode | null;
 	inlineNamespace: NamespaceVariable | null = null;
+	get isFollowingTopLevelAwait(): boolean {
+		return isFlagSet(this.flags, Flag.isFollowingTopLevelAwait);
+	}
+	set isFollowingTopLevelAwait(value: boolean) {
+		this.flags = setFlag(this.flags, Flag.isFollowingTopLevelAwait, value);
+	}
```

Comment:
I wonder if we really need a bit-flag here as it is rather trivial to calculate on the fly. What if instead we have a method

```ts
getIsFollowingTopLevelAwait(): boolean {
  return this.parent instanceof AwaitExpression && this.parent.isTopLevelAwait
}
```

and call that from `Module.includeDynamicImport`? I think in the current code, it will only ever be called once for every dynamic import on first inclusion. But this also removes an indirect dependency where we need to make sure the `ImportExpression` is included before the dynamic import is included.

Reply:
That makes sense! Done.